File: backend/models/certification.go
package models

type Certification struct {
	ID          string `json:"id"`
	Name        string `json:"name"`
	Description string `json:"description"`
	Range       string `json:"range"`
	ImgID       string `json:"imgID"`
}

type CertificationComplete struct {
	ID string `json:"id"`
}

File: backendv2/backend/backend/Schemas/Certification.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace backend.Schemas
{
	public class Certification
	{
        [Required]
        public string ID { get; set; } = string.Empty;

        [Required]
        public string Name { get; set; } = string.Empty;

        [Required]
        public string Description { get; set; } = string.Empty;

        [Required]
        public string Range { get; set; } = string.Empty;

        [Required]
        public string ImgID { get; set; } = string.Empty;

    }
}

